website/content/en/status/report-2024-01-2024-03/cloud-init.adoc

```diff
@@ -0,0 +1,34 @@
+=== FreeBSD as a Tier 1 cloud-init Platform
+
+Links: +
+link:https://cloud-init.io/[cloud-init Website] URL: link:https://cloud-init.io/[] +
+link:https://cloudinit.readthedocs.io/en/latest/[cloud-init Documentation] URL: link:https://cloudinit.readthedocs.io/en/latest/[] +
+
+Contact: Mina Galić <freebsd@igalic.co>
+
+cloud-init is the standard way of provisioning servers in the cloud.
+Over the past year and a half, thanks to this FreeBSD support has steadily improved.
+This year, we took a pivot and decided to explicitly focus on making improvements in FreeBSD itself, that will aid the cloud-init team to test future changes to FreeBSD code-paths themselves.
+To achieve this goal, we need to make FreeBSD run in LXD (and Incus), which it already can, but not under control of ``lxd-agent`` (or ``incus-agent``).
+
+Here are some improvements from the recent weeks:
+
+- I have written a small https://codeberg.org/meena/test-cloud-init[testing-framework] (in sh, and I'm slowly porting it to OpenTofu/Terraform), which installs the latest version of package:net/cloud-init-devel[] or package:net/cloud-init[] and runs a couple of standard cloud-init tests.
+- To do this, I have created a https://pkg.igalic.co/[dedicated public repository] which contains the latest versions of package:net/cloud-init-devel[] and package:net/cloud-init[] for FreeBSD 13 and 14 on amd64 and aarch64
+- I have ported Linux' vsock testing framework to https://codeberg.org/meena/vsock-tests[FreeBSD]
+- Created the baseline of a https://codeberg.org/meena/freebsd-src/src/branch/vsock/sys/dev/virtio/socket[VirtIO Socket driver], based on the HyperV Socket driver.
+- In doing so, I made numerous https://reviews.freebsd.org/D44517[improvements to HyperV sockets], some of which are accepted, others still need more work.
+- I have tested and released the latest 24.1 series cloud-init, where we have finally fixed some longstanding bugs, such as [moving ``/run/cloud-init``]https://github.com/canonical/cloud-init/pull/4820 to ``/var/run/cloud-init`` on BSD, as well as fixing the https://github.com/canonical/cloud-init/pull/5061[``homedir`` argument] to ``user_groups`` to actually do something.
+- This release also sees numerous fixes to the OpenBSD code-paths from the community and not just me.
+- I've also started an official port for OpenBSD, but that work https://marc.info/?l=openbsd-ports&m=170508174230708&w=2[has stalled].
+
+The work to come, in broad strokes:
+
+- finish the FreeBSD VirtIO Socket driver
+- fix Go's runtime to support VirtIO on FreeBSD
+- port lxd-agent's dependencies to FreeBSD
+- port lxd-agent to FreeBSD
```

Comment:
Please be consistent with your lists: the first list has all elements starting with uppercase letters, this one has all elements starting with lowercase letters. Also the first list has all elements (except one) finishing with a period, while all  the elements of this list end with no punctuation.